src/Database/Data/ArtifactData.ts

```diff
@@ -77,8 +77,17 @@ export class ArtifactDataManager extends DataManager<string, string, ICachedArti
         this.database.chars.setEquippedArtifact(newChar.key, slotKey, newArt.id)
       if (prevChar)
         this.database.chars.setEquippedArtifact(prevChar.key, slotKey, prevArt?.id ?? "")
-    } else if (newArt.location) // Trigger a update to character as well
+    } else if (newArt.location) {
+      const char = this.database.chars.get(newArt.location)
```

Comment:
This block would be invoked unnecessarily if `oldArt` exists, but also has the same location as `newArt`